packages/lisk-transactions/src/transactions/3_vote_transaction.ts

```diff
@@ -107,7 +106,6 @@ const validateInputs = ({
 export class VoteTransaction extends BaseTransaction {
 	public readonly containsUniqueData = true;
 	public readonly asset: VoteAsset;
-	public readonly fee: BigNum = new BigNum(VOTE_FEE);
 
 	public constructor(tx: TransactionJSON) {
```

Comment:
we need to add assignment of the `_fee` in constructor

Reply:
Addressed.